contributing_to_docs/doc_guidelines.adoc

```diff
@@ -13,25 +13,21 @@ for guidelines for OpenShift / OKD 3.x documentation.
 
 toc::[]
 
-== Topic Metadata
-Every topic file should contain the following metadata at the top, with no line spacing in between, except where noted:
+== Topic/Assembly Metadata
+Every topic file should contain the following metadata at the top, with no line
+spacing in between, except where noted:
 
 ----
-= Document/Topic Title                                          <1>
-:data-uri:                                                      <2>
-:icons:                                                         <3>
-:toc: macro                                                     <4>
-:toc-title:                                                     <5>
------------intentional blank line--------------
-toc::[]                                                         <6>
+[id='<unique-heading-for-asssembly>']                           <1>
+= Document/Topic Title                                          <2>
```

Comment:
We use sentence case, right? So maybe this example should as well.

Reply:
The whole guideline uses Title case!! I have done updates throughout.